deployment/hasura/metadata/actions.yaml

```diff
@@ -79,6 +79,13 @@ actions:
     permissions:
       - role: aerie_admin
       - role: user
+  - name: getActivityEffectiveArgumentsBulk
+    definition:
+      kind: ""
+      handler: "{{AERIE_MERLIN_URL}}/getActivityEffectiveArgumentsBulk"
+      timeout: 300
+    permissions:
+      - role: user
```

Comment:
Changing out from under you: https://github.com/NASA-AMMOS/aerie/pull/1028
Can we add `- role: aerie_admin` here as well?